src/pylast/__init__.py

```diff
@@ -774,40 +771,38 @@ def __init__(
             },
         )
 
-    def __repr__(self):
-        return "pylast.LibreFMNetwork(%s)" % (
-            ", ".join(
-                (
-                    "'%s'" % self.api_key,
-                    "'%s'" % self.api_secret,
-                    "'%s'" % self.session_key,
-                    "'%s'" % self.username,
-                    "'%s'" % self.password_hash,
-                )
-            )
+    def __repr__(self) -> str:
+        return (
+            "pylast.LibreFMNetwork("
+            f"'{self.api_key}', "
+            f"'{self.api_secret}', "
+            f"'{self.session_key}', "
+            f"'{self.username}', "
+            f"'{self.password_hash}'"
+            ")"
         )
 
 
 class _ShelfCacheBackend:
     """Used as a backend for caching cacheable requests."""
 
-    def __init__(self, file_path=None, flag=None):
+    def __init__(self, file_path=None, flag=None) -> None:
         if flag is not None:
             self.shelf = shelve.open(file_path, flag=flag)
         else:
             self.shelf = shelve.open(file_path)
         self.cache_keys = set(self.shelf.keys())
 
-    def __contains__(self, key):
+    def __contains__(self, key) -> bool:
         return key in self.cache_keys
 
-    def __iter__(self):
+    def __iter__(self) -> Iterator:
```

Comment:
`Iterator` should be subscripted, to indicate the type of the values returned by the iterator.

Reply:
It's something like `Iterator[typing.KeysView[builtins.str*]]` so I'm just going to skip this for now.

Reply:
It sounds like `Iterator[str]` would suffice.

You could always add a type alias though.

Reply:
Thanks, will leave it for now and get a release out. Still plenty more typing that can be added later.